deps/db/src/logseq/db.cljs

```diff
@@ -435,12 +438,30 @@
     (:alias rules/rules))
    distinct))
 
+(defn page-alias-set
+  [db page-id]
+  (->>
+   (get-block-alias db page-id)
+   (set)
+   (set/union #{page-id})))
+
 (defn get-block-refs
   [db id]
-  (let [alias (->> (get-block-alias db id)
+  (let [entity (d/entity db id)
+        alias (->> (get-block-alias db id)
                    (cons id)
                    distinct)
-        refs (->> (mapcat (fn [id] (:block/_path-refs (d/entity db id))) alias)
+        refs (->> (mapcat (fn [id]
+                            (->> (:block/_refs (d/entity db id))
+                                 (remove (fn [ref]
+                                           ;; remove refs that have the block as either tag or property
```

Comment:
File graphs are no longer showing properties or tags as linked refs. I think this may be the cause

Reply:
Fixed by https://github.com/logseq/logseq/commit/43e9f9707c98a44400c768d3a78ba17da3cc9fef.

Reply:
Confirm fixed. At some point it would be helpful if logseq.db was more organized with db and file only fns in different namespaces. I wasn't able to notice this by reading the code but only by accident in QA